js/symbol/collision_tile.js

```diff
@@ -28,6 +31,17 @@ function CollisionTile(angle, pitch) {
     // The amount the map is squished depends on the y position.
     // Sort of account for this by making all boxes a bit bigger.
     this.yStretch = Math.pow(this.yStretch, 1.3);
+
+    this.edges = [
```

Comment:
Do we need to create an instance of `edges` per tile? Couldn't there just be one global instance?